File: app/src/main/java/org/jay/launchstarterexample/tasks/InitStethoTask.java
package org.jay.launchstarterexample.tasks;

import android.os.Handler;
import android.os.Looper;
import com.facebook.stetho.Stetho;
import org.jay.appstarter.Task;

/**
 * 异步的Task
 */
public class InitStethoTask extends Task {

    @Override
    public void run() {

        Handler handler = new Handler(Looper.getMainLooper());
        Stetho.initializeWithDefaults(mContext);
    }
}

File: app/src/main/java/org/jay/appstarterexample/tasks/InitStethoTask.kt
package org.jay.appstarterexample.tasks

import android.os.Handler
import android.os.Looper
import com.facebook.stetho.Stetho
import org.jay.appstarter.Task

/**
 * 异步的Task
 */
class InitStethoTask : Task() {

    override fun run() {
        Stetho.initializeWithDefaults(mContext)
    }
}